modules/verifying-serverless-app-deployment.adoc

```diff
@@ -24,3 +31,34 @@ helloworld-go   http://helloworld-go.default.example.com   helloworld-go-4wsd2
 $ curl http://helloworld-go.default.example.com
 Hello World: Go Sample v1!
 ----
++
+----
+$ curl https://helloworld-go.default.example.com
+Hello World: Go Sample v1!
+----
+
+== Self signed certificates
+
+If you receive an error relating to a self-signed certificate in the certificate chain, you can add the ``--insecure` flag to the curl command to ignore the error.
+
+[IMPORTANT]
+====
+Self-signed certificates must not be used in a production deployment. This method is only for testing purposes.
+====
+
+.Example
+----
+$ curl https://helloworld-go.default.example.com --insecure
+Hello World: Go Sample v1!
+----
+
+== Certificate authority signed certificates
+
+If your {product-title} cluster is configured with a certificate that is signed by a certificate authority (CA) but not yet globally configured for your system,  you must specify this with the curl command.
```

Comment:
There's an extra space after comma but otherwise these changes look good to me.

Reply:
Fixed now, can I get a `QE Approved` please? :slightly_smiling_face: thanks!